### PR TITLE
Implementing some of the VDR feedback

### PR DESCRIPTION
**Pull request by acostadon**
Adding some content and navigation options per VDR,
resolves  https://github.com/rapidsai/graph_dl/issues/594
resolves https://github.com/rapidsai/graph_dl/issues/593

### CODE REVIEW

**Review by BradReesWork — APPROVED** [review]

**Comment by nv-rliu** [comment]
Left some minor suggestions. Thanks

File: docs/cugraph/source/index.rst [hunk 1]
```diff
@@ -1,58 +1,84 @@
...
+  * `Docker <https://docs.rapids.ai/install#wsl-docker>`_
+  * `pip <https://docs.rapids.ai/install#wsl-pip>`_
 
-Note: We are redoing all of our documents, please be patient as we update
-the docs and links
 
-|
+Build From Source
+
+To build from source, check each RAPIDS GitHub README for set up and build instructions. Further links are provided in the `selector tool <https://docs.rapids.ai/install#selector>`_.
```
Comment (nv-rliu) [inline docs/cugraph/source/index.rst:49]: > check each RAPIDS GitHub README for set up and build instructions

Is this referring to each of the separate Graph projects' repos in preparation for the migration?

Also, I don't think the install selector tool is related to building from source.

**Review by nv-rliu — APPROVED** [review]

**Comment by acostadon** [comment]
rerun tests

**Review by rlratzel — APPROVED** [review]

**Pull request merged**